### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by giuseppe**

### CODE REVIEW

File: keps/sig-node/127-user-namespaces/README.md [hunk 1]
```diff
@@ -603,6 +641,15 @@ use container runtime versions that have the needed changes.
...
 ### Upgrade / Downgrade Strategy
 
+Existing pods will still work as intended, as the new field is missing there.
+
+Upgrade will not change any current behaviors.
+
+When the new functionality wasn't yet used, downgrade will not be affected.
+
+Versions of Kubernetes that doesn't have this feature implemented will
+ignore and strip out the new field `pod.spec.hostUsers`.
```
Comment (rata) [inline keps/sig-node/127-user-namespaces/README.md]: The object won't be modified, right? Reviews where picky on the words, so I would leave this to just "ignore the new field..."

File: keps/sig-node/127-user-namespaces/README.md [hunk 2]
```diff
@@ -635,11 +682,12 @@ doesn't create them. The runtime can detect this situation as the `user` field
 in the `NamespaceOption` will be seen as nil, [thanks to
 protobuf][proto3-defaults]. We already tested this with real code.
 
-Old runtime and new kubelet: containers are created without userns. As the
-`user` field of the `NamespaceOption` message is not part of the runtime
-protofiles, that part is ignored by the runtime and pods are created using the
-host userns.
+Old runtime and new kubelet: the runtime won't report that it supports
+user namespaces through the `StatusResponse` message, so the kubelet
+will detect it and fail when such a request is done.
```
Comment (rata) [inline keps/sig-node/127-user-namespaces/README.md]:
```suggestion
will detect it and return an error if a pod with user namespaces is created.
```

File: keps/sig-node/127-user-namespaces/README.md [hunk 3]
```diff
@@ -603,6 +641,15 @@ use container runtime versions that have the needed changes.
 
 ### Upgrade / Downgrade Strategy
 
+Existing pods will still work as intended, as the new field is missing there.
+
+Upgrade will not change any current behaviors.
+
+When the new functionality wasn't yet used, downgrade will not be affected.
```
Comment (rata) [inline keps/sig-node/127-user-namespaces/README.md:648]: I would add what happens if the functionality was used. Something like:

On downgrade, when the functionality was used, the pods created with user namespaces that are running will continue to run with user namespaces. Pods will need to be re-created to stop using the user namespace.

I'm not sure what will happen if a pod is _restarted_, like it was started with userns just fine, the kubelet is restarted to disable the feature and the pod is restarted by the kubelet. Will it use user namespaces or not? :thinking: 

I think we should mention that case too. or is it obvious to you what will happen?

File: keps/sig-node/127-user-namespaces/README.md (hunk 1 above)
Comment (giuseppe) [reply keps/sig-node/127-user-namespaces/README.md]: I'll fix it

**Pull request closed without merging**